testing/testcafe/tests/dataGrid/editing.ts

```diff
@@ -2185,3 +2185,28 @@ test('Cells should be focused correctly on click when cell editing mode is used
       },
     }));
   });
+
+test('Popup EditForm screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+  const commandCellRow0 = dataGrid.getDataCell(0, 2);
+
+  await t
+    .click(commandCellRow0.getLinkEdit())
+    // act
+    .expect(await takeScreenshot('popup-edit-form', dataGrid.element))
```

Comment:
It's better to add the extension to the screenshot name:

```suggestion
    .expect(await takeScreenshot('popup-edit-form.png', dataGrid.element))
```